File: src/main/java/org/example/bankramenserver/global/jwt/JwtProperties.java
package org.example.bankramenserver.global.jwt;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "jwt")
public class JwtProperties {

    private String secretKey;
    private String header;
    private String prefix;
    private long accessExp;
    private long refreshExp;
}

File: src/main/kotlin/org/example/bankramenserver/global/jwt/JwtProperties.kt
package org.example.bankramenserver.global.jwt

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "jwt")
class JwtProperties { var secretKey: String? = null; var header: String? = null; var prefix: String? = null; var accessExp: Long = 0; var refreshExp: Long = 0 }
